migrator/migrations/m_179_to_m_180_openshift_policy_exclusions/policies_before_and_after/after/host_network.json

```diff
@@ -0,0 +1,247 @@
+{
+  "id": "6226d4ad-7619-4a0b-a160-46373cfcee66",
+  "name": "Docker CIS 5.9 and 5.20: Ensure that the host's network namespace is not shared",
+  "description": "When HostNetwork is enabled the container is not placed inside a separate network stack. The container's networking is not containerized when this option is applied. The consequence of this is that the container has full access to the host's network interfaces. It also enables a shared UTS namespace. The UTS namespace provides isolation between two system identifiers: the hostname and the NIS domain name. It is used to set the hostname and the domain which are visible to running processes in that namespace. Processes running within containers do not typically require to know either the hostname or the domain name. The UTS namespace should therefore not be shared with the host.",
+  "rationale": "Selecting this option is potentially dangerous. It allows the container process to open reserved low numbered ports in the way that any other root process can. It also allows the container to access network services such as D-bus on a Docker host. A container process could potentially carry out undesired actions, such as shutting down the host. The container will also share the network namespace with the host, providing full permission for each container to change the hostname of the host. This is not in line with good security practice and should not be permitted.",
+  "remediation": "You should not create a deployment with `hostNetwork: true`",
+  "categories": [
+    "Docker CIS"
+  ],
+  "lifecycleStages": [
+    "DEPLOY"
+  ],
+  "exclusions": [
+    {
+      "name": "Don't alert on deployment gcp-pd-csi-driver-controller in namespace openshift-cluster-csi-drivers",
+      "deployment": {
+        "name": "gcp-pd-csi-driver-controller",
+        "scope": {
+          "namespace": "openshift-cluster-csi-drivers"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment gcp-pd-csi-driver-node in namespace openshift-cluster-csi-drivers",
+      "deployment": {
+        "name": "gcp-pd-csi-driver-node",
+        "scope": {
+          "namespace": "openshift-cluster-csi-drivers"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment machine-approver in namespace openshift-cluster-machine-approver",
```

Comment:
I don't see this exclusion for this policy (Docker CIS 5.9 and 5.20: Ensure that the host's network namespace is not shared)

Reply:
it is added in row 6